pkg/cli/cmd/pipelinerun/list.go

```diff
@@ -239,10 +237,10 @@ List PipelineRuns with partial pipeline name match:
 		},
 	}
 
-	cmd.Flags().Int32VarP(&opts.Limit, "limit", "l", 10, "Maximum number of PipelineRuns to return")
+	cmd.Flags().Int32VarP(&opts.Limit, "limit", "l", 50, "Maximum number of PipelineRuns to return (must be between 5 and 100 and defaults to 50)")
```

Comment:
based on the actual code and discussion
```suggestion
	cmd.Flags().Int32VarP(&opts.Limit, "limit", "l", 50, "Maximum number of PipelineRuns to return (must be between 5 and 1000, default is 50)")
```

Reply:
Ah yes, I missed changing it here.

Reply:
Fixed.